### DISCUSSION TITLE
How to do Resumeable Uploads to Supabase storage with python?

### DISCUSSION THREAD

**Discussion by belliedmonkey** [supabase/supabase#22090] [Questions]
How to do Resumeable Uploads with python?

**Comment by silentworks** [comment]
This feature hasn't been added to the library as yet. I'm still investigating how to get his done correctly in Python.

**Reply by silentworks** [reply]
Looks like you should be able to use the `tus` python client to do resumable uploads with Python https://github.com/tus/tus-py-client. It would follow similar steps to this article https://supabase.com/docs/guides/storage/uploads/resumable-uploads?language=js, if you are having any issue let me know and I will see how best I can help.
